README.md

```diff
@@ -38,8 +38,10 @@ The data gathering workflow is as follows:
   data to the breaking update JSON format:
   ```json
   {
-    "labels": ["COMPILATION_FAILURE", "TEST_FAILURE", "PRECEDING_COMMIT_COMPILATION_FAILURE", 
-               "PRECEDING_COMMIT_COMPILATION_FAILURE", "NO_FAILURE"], 
+    "labels": ["DEPENDENCY_RESOLUTION_FAILURE", "MAVEN_ENFORCER_ERROR", "COMPILATION_FAILURE", 
+               "TEST_FAILURE", "UNKNOWN_FAILURE", "PRECEDING_COMMIT_DEPENDENCY_RESOLUTION_FAILURE", 
+               "PRECEDING_COMMIT_MAVEN_ENFORCER_ERROR", "PRECEDING_COMMIT_COMPILATION_FAILURE", 
+               "PRECEDING_COMMIT_TEST_FAILURE", "PRECEDING_COMMIT_UNKNOWN_FAILURE", "NO_FAILURE"], 
```

Comment:
we can rename "NO_FAILURE" into "SUCCESS"

Reply:
The label "NO_FAILURE" indicates that there were no failure before or after updating the dependency. As this is an unsuccessful reproduction, this will no longer be saved according to https://github.com/chains-project/breaking-updates/issues/53. Therefore, this label will be removed.